.github/workflows/ci.yml

```diff
@@ -189,3 +189,19 @@ jobs:
         with:
           name: avt
           path: .playwright/report
+
+  build-components-json:
```

Comment:
Would it satisfy requirements if this was uploaded as an artifact when something is merged into `main`?

Reply:
It's nice to see the `generated/component.json` diff in the same PR that made the changes. I don't think we get that if we only run this when merging to `main`, correct?